test/org/sagebionetworks/bridge/models/ClientInfoTest.java

```diff
@@ -118,6 +119,14 @@ public void longFormCorrectlyParsed() {
         assertEquals("iPhone OS 9.0.2", info.getOsVersion());
         assertEquals("BridgeSDK", info.getSdkName());
         assertEquals(4, info.getSdkVersion().intValue());
+        
+        info = ClientInfo.parseUserAgentString(VALID_LONG_UA_3);
+        assertEquals("Belgium", info.getAppName());
+        assertEquals(2, info.getAppVersion().intValue());
+        assertEquals("Motorola Flip-Phone", info.getOsName());
+        assertEquals("Android 14", info.getOsVersion());
+        assertEquals("BridgeJavaSDK", info.getSdkName());
+        assertEquals(10, info.getSdkVersion().intValue());
```

Comment:
I saw this string generated in the SDK and had to verify it's valid in the server tests. It's valid.